examples/involvingstates/example_state_exclusion.py

```diff
@@ -0,0 +1,268 @@
+"""Quantum state exclusion
+===========================
+
+In this tutorial, we are going to cover the problem of *quantum state
+exclusion*. We are going to briefly describe the problem setting and then
+describe how one may use :code:`|toqito⟩` to calculate the optimal probability
+with which this problem can be solved for a number of different scenarios.
+"""
+# sphinx_gallery_thumbnail_path = 'figures/trine.png'
```

Comment:
Same as my other comment, this appears in the build. 

https://toqito--1252.org.readthedocs.build/en/1252/auto_examples/involvingstates/example_state_exclusion.html